test/cpp/thread_manager/thread_manager_test.cc

```diff
@@ -91,7 +91,7 @@ grpc::ThreadManager::WorkStatus ThreadManagerTest::PollForWork(void **tag,
   // Simulate "polling for work" by sleeping for sometime
   SleepForMs(kPollingTimeoutMsec);
 
-  *tag = nullptr;
+  *tag = NULL;
```

Comment:
The symbol `nullptr` is not being recognized since it was added in gcc4.6.  So I am not sure if `static_cast<void *>(nullptr)` would help :(.  
Ideally doing `#include grpc++/impl/codegen/config.h` should have helped (and looks like that is what we do at other places) but I am not sure why...